### ISSUE TITLE
Please, clarify the copyholders names

### ISSUE THREAD

**Issue by eribertomota** [andilippi/obs-scene-as-transition#8]
Hi @andilippi,

Initially, thanks and congratulations for your first work.

I intend to package this plugin for Debian (a big Linux distribution). When checking the licensing, I found two files quoting Exeldro (resource.rc.in and scene-as-transition.rc). I know Exeldro and I maintain some plugins from him on Debian. I also know that he helped you to develop this plugin. My question is if the Exeldro's name in these files were a mistake. I need to know about this to generate the debian/copyright file correctly.

Thanks in advance.

**Comment by andilippi** [comment]
Hey there! 

Thanks for reaching out. When Exeldro was helping me get started with building the plugin we used one of his plugins as the template. I think we just forget to change his name on those files! Haha I'm still new!

**Comment by eribertomota** [comment]
I will send a pt-BR translation in some minutes. Could you fix the copyholder name and release a new version?

**Comment by eribertomota** [comment]
The pt-BR was sent. I also noticed that scene-as-transition.h is empty. Trash?

**Comment by andilippi** [comment]
Sorted

**Comment by eribertomota** [comment]
This is for informational purposes only. 

Copyright issues was fixed via 46c240d and a30b061.
Empty file was fixed via 453be69.
FTBFS will be fixed via #10

I just uploaded the version 1.1.0+20230419.ce423a3 to Debian. Considering this is the first upload to Debian, the package needs to be inspected by FTP-Master Team. The time to make this action is from 1 day and up to 120 days. The package will wait in the New Queue[1].

[1] https://ftp-master.debian.org/new.html
